packages/angular/src/lib/injection.tokens.ts

```diff
@@ -0,0 +1,25 @@
+import { InjectionToken } from '@angular/core';
+
+export const NODE_MODEL: InjectionToken<string> = new InjectionToken(
```

Comment:
Maybe simplify and have one MODEL injectionToken instead per type

Reply:
wouldn't that might introduce issues if trying to inject different models in custom implementations?